deltachat-ios/Coordinator/AppCoordinator.swift

```diff
@@ -861,32 +849,6 @@ class EditGroupCoordinator: Coordinator {
     }
 }
 
-// MARK: - EditContactCoordinator
-class EditContactCoordinator: Coordinator, EditContactCoordinatorProtocol {
-    var dcContext: DcContext
-    let navigationController: UINavigationController
-
-    var childCoordinators: [Coordinator] = []
-
-    init(dcContext: DcContext, navigationController: UINavigationController) {
-        self.dcContext = dcContext
-        self.navigationController = navigationController
-    }
-
-    func navigateBack() {
-        navigationController.popViewController(animated: true)
-    }
-
-    func showChat(chatId: Int) {
-        let chatViewController = ChatViewController(dcContext: dcContext, chatId: chatId)
-        let coordinator = ChatViewCoordinator(dcContext: dcContext, navigationController: navigationController, chatId: chatId)
-        coordinator.chatViewController = chatViewController
-        childCoordinators.append(coordinator)
```

Comment:
i think, this was wrong anyway, why append a coordinator that is just discarded to the list of childCoordinators?